environments/nanopore/pangolin/environment_pangolin.yml

```diff
@@ -10,10 +10,9 @@ dependencies:
   - python>=3.7
   - snakemake-minimal>=5.13
   - gofasta
-  - usher
+  - usher>=0.3.2
   - pip:
     - pandas==1.0.1
-    - git+https://github.com/cov-lineages/pangolin.git
```

Comment:
NP pipe doesn't require pangolin?

Reply:
pangolin is added as a separate container for the nanopore. It has some conflicts if I try to put them together.